chore: updating pubsub system (#3646)
open-policy-agent/gatekeeper: website/docs/audit.md

All of these events (including `violation_audited`) are marked 
with the same `audit_id` for a given audit run.

### Export violations

> This feature allows plugging in different backends that allows Gatekeeper to export audit violations. Therefore, violations are not subject to reporting limits. Please refer to [this](export.md) guide to configure audit to push violations via this feature.

Reviewer: ```suggestion
This feature allows different systems to be provided for Gatekeeper to export audit violations. Thereby, ensuing violations are not subject to reporting limits. Please refer to [this](export.md) guide to configure audit to push violations via this feature.
```

> This feature allows different systems to be provided for Gatekeeper to export audit violations. Thereby, ensuing violations are not subject to reporting limits. Please refer to [this](export.md) guide to configure audit to push violations via this feature.

Reviewer: ```suggestion
This feature allows different systems to be provided for Gatekeeper to export audit violations. Thereby, ensuring violations are not subject to reporting limits. Please refer to [the export guide](export.md) to configure audit to push violations via this feature.
```

Author: Updated the docs.

Limitations/drawbacks of exporting violations:

- There is a risk of messages getting dropped. You might not receive all the exported violations. This depends on the type of backend you are using for delivery. For example, using a network as backend to export violation has the risk of messages getting dropped.

> - Additional dependency depending on what is plugged in. For example, using pubsub tools to export violations.

Reviewer: ```suggestion
- Additional dependency on the backend system provided. For example, using pubsub tools to export violations.
```

## Running Audit
For more details on how to deploy audit and 
number of instances to run, please refer to [operations audit](operations.md#audit). 

## Configuring Audit

> - Audit violations per constraint: set `--constraint-violations-limit=123` (defaults to `20`). NOTE: This flag only impacts when gathering audit results using the constraint status model. If you want to export audit results via other means, please refer to the [exporting audit results](export.md) guide. Both approaches for getting audit violations can be configured independently and work simultaneously without any interference.

Reviewer: ```suggestion
- Audit violations per constraint: set `--constraint-violations-limit=123` (defaults to `20`). NOTE: This flag only impacts when gathering audit results using the constraint status model. If you want to export audit results via other means, please refer to [the export guide](export.md). Both approaches for getting audit violations can be configured independently and work simultaneously without any interference.
```

> - Audit violations per constraint: set `--constraint-violations-limit=123` (defaults to `20`). NOTE: This flag only impacts when gathering audit results using the constraint status model. If you want to gather audit results via other means, please refer to the [exporting audit results](export.md) guide. Both approaches for getting audit violations can be configured independently and work simultaneously without any interference.

Reviewer: ```suggestion
- Audit violations per constraint: set `--constraint-violations-limit=123` (defaults to `20`). NOTE: This flag only impacts when gathering audit results using the constraint status model. If you want to export audit results via other means, please refer to the [exporting audit results](export.md) guide. Both approaches for getting audit violations can be configured independently and work simultaneously without any interference.
```
